new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonUtils.java

```diff
@@ -0,0 +1,655 @@
+
+    logger.info("ingress {0} for domain {1} was created in namespace {2}",
+        ingressName, domainUid, domainNamespace);
+
+    return ingressHostList;
+  }
+
+  /**
+   * Check pod is created.
+   *
+   * @param podName pod name to check
+   * @param domainUid the domain in which the pod exists
+   * @param domainNamespace the domain namespace in which the domain exists
+   */
+  public static void checkPodCreated(String podName, String domainUid, String domainNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podExists(podName, domainUid, domainNamespace),
+            String.format("podExists failed with ApiException for pod %s in namespace %s",
+                podName, domainNamespace)));
+  }
+
+  /**
+   * Check pod is ready.
+   *
+   * @param podName pod name to check
+   * @param domainUid the domain in which the pod exists
+   * @param domainNamespace the domain namespace in which the domain exists
+   */
+  public static void checkPodReady(String podName, String domainUid, String domainNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be ready in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> podReady(podName, domainUid, domainNamespace),
+            String.format("podReady failed with ApiException for pod %s in namespace %s",
+               podName, domainNamespace)));
+  }
+
+  /**
+   * Check service is created.
+   *
+   * @param serviceName service name to check
+   * @param domainNamespace the domain namespace in which the service exists
+   */
+  public static void checkServiceCreated(String serviceName, String domainNamespace) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for service {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                serviceName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(assertDoesNotThrow(() -> serviceExists(serviceName, null, domainNamespace),
+            String.format("serviceExists failed with ApiException for service %s in namespace %s",
+                serviceName, domainNamespace)));
+  }
+
+  /**
+   * Check pod was deleted.
```

Comment:
you are not checking it was deleted, you are just checking it does not exist

Reply:
fixed